### ISSUE TITLE
Question: Is this compatible with the other fix?

### ISSUE THREAD

**Issue by kik4444** [z0z0z/mf-installcab#5]
So this one is Installcab based and it's been confirmed to work with at least 3 games, whereas the other fix (https://github.com/z0z0z/mf-install) is not Installcab based. However, if I apply this fix, does it also cover all the games from the other fix?

**Comment by z0z0z** [comment]
No, the other fix is for games where the first one doesn't work, like Resident Evil 2.

If you happen to apply both fixes to the same prefix, a game may still work, wouldn't recommend it though tbh.
